src/Ionide.ProjInfo/Library.fs

```diff
@@ -776,7 +776,7 @@ type WorkspaceLoaderViaProjectGraph private (toolsPath, ?globalProperties: (stri
                 )
 
                 let gbr =
-                    GraphBuildRequestData(projects, ProjectLoader.designTimeBuildTargets false, null, BuildRequestDataFlags.ReplaceExistingProjectInstance)
+                    GraphBuildRequestData(projects, ProjectLoader.designTimeBuildTargets false, null, BuildRequestDataFlags.ReplaceExistingProjectInstance ||| BuildRequestDataFlags.ClearCachesAfterBuild)
```

Comment:
is there a test case we need to add to verify this behavior? an extracted example that was definitely failing before?

Reply:
Added a test.  If you remove this change, it should fail for `WorkspaceLoaderViaProjectGraph`